### DISCUSSION TITLE
Import Everything issue

### DISCUSSION THREAD

**Discussion by Spivoran** [requarks/wiki#6128] [Help / Questions]
Hello,

I'm running the Digital Ocean Wiki.JS droplet with local storage enabled. After archiving  my pages (with `Dump all content to disk`) and shutting down an old instance of the wiki, I want to get it going again. I placed the 2450 archived pages into local storage and hit the `Import Everything` button. At first it seemed to all work great, but after some scrutiny, it has come to my attention that a few hundred pages didn't import properly. Each page looks just like the archived page with title, description, tags, etc. between two sets of three dashes at the head of the file, and none of those attributes were saved to the page's properties. Each page's title is instead, the filename of the source file. Of course, I could fix them manually, but it's a big job. I have a couple of questions:

1) Did I do something wrong during the import process to goof it up? The files were unchanged from the time they were downloaded after the dump.
2) Can I just make another attempt to `Import Everything` and see if it works this time, even though everything has been imported once?
3) If I do this, will I end up with duplicate pages (and assets)?
4) I could take some time to catalogue all of the botched page imports and attempt only to `Import Everything` on those pages alone, after removing everything else from the local storage folder. This would take more effort, but not as much as fixing them manually.

One interesting side note: When attempting to search to find which pages were part of the problem, I tried to use the search function to locate all files with the string `dateCreated` in it, but no results were returned. It seems like anything in the 'header' of the file is not indexed for searching.

Thanks for reading,
S

**Comment by NGPixel** [comment]
Look at the logs during the import. Some pages could have broken formatting that prevents the rendering from completing / reading the metadata correctly.

**Reply by Spivoran** [reply]
I stopped and restarted the wiki container to perform LetsEncrypt initialization after I imported everything, and `docker logs wiki` from the host does not provide any information going back before this restart. Is there another way to access logs that would go back before the container was restarted?

**Reply by NGPixel** [reply]
Run `docker logs -f wiki` before restarting the container
